recipe/build.sh

```diff
@@ -0,0 +1,6 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+
+python setup.py install --old-and-unmanageable
```

Comment:
I just noticed that in `.circleci/config.yml` you removed the `--old-and-unmanageable`, was it indeed your intention to add/keep it here?